### DISCUSSION TITLE
Birt Report not working without internet

### DISCUSSION THREAD

**Discussion by arunssasidhar** [eclipse-birt/birt#1198] [Q&A]
Hi,

We are using Birt report in our Java application running in Tomcat. We use Oracle Linux 7.9 as OS for the application server (Minimal installation without Desktop). We are facing an issue with the report. Whenever the Internet goes, the Birt report stops working. We don't see any error in the logs. The report just hung.

Could someone help us to identify and fix the issue?

Thanks and regards,
Arun

**Comment by wimjongman** [comment]
Open the report design in notepad and scan for any http: https: code.

**Reply by hvbtup** [reply]
This is probably not a BIRT issue. Probably, your report, a library used by the report, or the JDBC driver tries to connect to the internet, for example when it tries to check for certificates or certificate revocation.

**Reply by hvbtup** [reply]
If you are using MSSQL server, this might be directly related to #1199.

**Comment by TheLaserNinja** [comment]
The Eclipse BIRT libraries don't do any external communications unless the report has it scripted. Some of the commercial offerings do license checks. I'd check what imports are in your app code.

**Comment by arunssasidhar** [comment]
Hi,

Thank you all for the input. The strange thing is that the same report works in Centos 7 without the internet. 

Thanks,
Arun Sasidharan
